### PR TITLE
[WIP] Radial menus for RCDs

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
:cl: Naksu
add: RCDs now have a comfier interface
/:cl:

This is still WIP, i want to put the other settings in this as well and maybe find better icons, also to ask antur how he solved the multiple menu issue

ezgif-5-47f60110c0

### CODE REVIEW

**Comment by coiax** [comment]
Cool!

Although I think you need to find a better way of describing "destroy" via iconography, since it just looks like there's some sort of RCD self destruct mode if it's on fire.

**Comment by Ratvar** [comment]
Chasm sprite, maybe?

**Comment by SgtxRootbeer** [comment]
Use a space tile for destroy? It's what you'll get if you over-use it anyways

**Comment by davethwave** [comment]
Maybe a half constructed door or something to show that the object is being deconstructed.

**Comment by GunHog** [comment]
What about a big red "X" on an airlock?

File: code/game/objects/items/RCD.dm [hunk 1]
```diff
@@ -361,19 +361,27 @@ RLD
...
+	var/list/choices = list(
+		"Airlock" = image(icon = 'icons/obj/doors/airlocks/station/public.dmi', icon_state = "closed"),
+		"Deconstruct" = image(icon= 'icons/mob/actions/actions_AI.dmi', icon_state = "detonate_rcds"),
+		"Grilles & Windows" = image(icon = 'icons/obj/structures_spawners.dmi', icon_state = "window_spawner"),
+		"Floors & Walls" = image(icon = 'icons/turf/floors.dmi', icon_state = "floor"),
+	)
+	choices["Airlock"].transform *= 0.65
+	choices["Grilles & Windows"].transform *= 0.65
+	choices["Floors & Walls"].transform *= 0.65
+	var/choice = show_radial_menu(user,src,choices)
```
Comment (Qustinnus) [inline code/game/objects/items/RCD.dm]: these are a thing??

**Comment by vuonojenmustaturska** [comment]
image
better placeholders

**Comment by MrDoomBringer** [comment]
what just use an eraser icon

**Comment by iskyp** [comment]
that door doesn't look like it feels so good

**Comment by coiax** [comment]
Can you make seperate icons for regular windows and reinforced windows?

**Comment by XDTM** [comment]
Maybe airlock access and color could be in there as well

**Comment by cacogen** [comment]
This is really good

**Pull request closed without merging**